sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -51,12 +52,16 @@
         this.service = service;
     }
 
-    Mono<Response<AnalyzeSentimentResult>> analyzeSentimentWithResponse(String text, String language, Context context) {
+    Mono<Response<com.azure.ai.textanalytics.models.DocumentSentiment>> analyzeSentimentWithResponse(
```

Comment:
can we import it?

Reply:
we already used "import com.azure.ai.textanalytics.**implementation**.models.DocumentSentiment;"
So can't import another same class name "com.azure.ai.textanalytics.models.DocumentSentiment"

Reply:
:| makes sense. Thanks for explaining